cfme/configure/configuration/diagnostics_settings.py

```diff
@@ -230,7 +230,7 @@ def update(self, updates, cancel=False, reset=False):
             flash_message = "Log Depot Settings were saved"
         else:
             logger.info('Settings were not updated')
-        view = self.create_view(flash_view, wait=10)  # implicit assert
+        view = self.create_view(flash_view, wait=20)  # implicit assert
```

Comment:
I think you don't need to increase the wait since you fixed `is_displayed`. Can you check it?

Reply:
Reset to default time.